packages/stack-shared/src/schema-fields.ts

```diff
@@ -267,7 +268,11 @@ export const userOAuthProviderSchema = yupObject({
   provider_user_id: yupString().required(),
 });
 export const userLastActiveAtMillisSchema = yupNumber().nullable().meta({ openapiField: { description: _lastActiveAtMillisDescription, exampleValue: 1630000000000 } });
-export const userOtpAuthEnabledSchema = yupBoolean().meta({ openapiField: { hidden: true, description: 'Whether the user has OTP/magic link enabled', exampleValue: true } });
+export const userOtpAuthEnabledSchema = yupBoolean().meta({ openapiField: { hidden: true, description: 'Whether the user has OTP/magic link enabled. ', exampleValue: true } });
+export const userOtpAuthEnabledMutationSchema = yupBoolean().meta({ openapiField: { hidden: true, description: 'Whether the user has OTP/magic link enabled. Note that only verified email can be used for sign-in with OTP. If OTP auth is not enabled for the project, it will throw an error.', exampleValue: true } });
+export const userHasPasswordSchema = yupBoolean().meta({ openapiField: { hidden: true, description: 'Whether the user has a password set. If the user does not have a password set, it will not be able to sign in with email/password.', exampleValue: true } });
+export const userPasswordSchema = yupString().meta({ openapiField: { description: 'A new password for the user, overwriting the old one (if it exists). Specifying this option revokes all current sessions.', exampleValue: 'my-new-password' } });
```

Comment:
```suggestion
export const userPasswordMutationSchema = yupString().meta({ openapiField: { description: 'Sets the user's password. Doing so revokes all current sessions.', exampleValue: 'my-new-password' } });
```

We should make it clear that this is for updating the password and not user passwords in general. Also, shouldn't this be nullable or so? (To disable password auth)?